### PR TITLE
Add May 17th extension banner to homepage

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

File: app/views/public_pages/home.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<% content_for :page_title do%>
-  <%=t("views.public_pages.home.title") %>
+<% content_for :page_title do %>
```
Comment (embarnard) [inline app/views/public_pages/home.html.erb:1]: A lot of these changes are just whitespace changes as a result of reformatting the file with cmd+opt+L

File: app/views/public_pages/home.html.erb [hunk 2]
```diff
@@ -38,6 +38,14 @@
     </div>
   </div>
 
+  <div class="slab banner">
```
Comment (embarnard) [inline app/views/public_pages/home.html.erb:41]: Actual addition here:

File: config/locales/es.yml [hunk 3]
```diff
@@ -1196,6 +1196,7 @@ es:
         subheader_offseason: GetYour Refund reanudará sus servicios en febrero para la temporada de impuestos 2021. Regístrese ahora, para ser notificado cuando empecemos nuestro servicio nuevamente.
         subheader: Maximice su reembolso al declarar sus impuestos de manera virtual, con nuestros profesionales certificados en impuestos.
         title: Ayuda gratuita para declarar sus impuestos ofrecida por voluntarios certificados por el IRS.
+        extension_banner_html: "<strong>Reminder: The federal income tax filing deadline has been extended to May 17, you can check your state filing deadline <a href='https://www.cnbc.com/2021/03/29/irs-extended-the-federal-tax-deadline-when-state-tax-returns-are-due.html' target='_blank' rel='noopener noreferrer'>here</a>.</strong> Unless you are filing a 2017 return or anticipate owing money this year, you can file until October 15th without a penalty. If you aren’t sure whether or not you will owe, you can complete and mail this <a href='https://www.irs.gov/pub/irs-pdf/f4868.pdf' target='_blank'>IRS form</a> requesting an extension."
```
Comment (embarnard) [inline config/locales/es.yml:1199]: Still waiting for Spanish translations

**Review by bytheway875 — APPROVED** [review]
This is good to go once you add the spanish translations!

**Pull request merged**